web/components/templates/dashboard/useDashboardPage.tsx

```diff
@@ -107,7 +118,7 @@ export const useDashboardPage = ({
     filterMap[modelFilterIdx] = {
       label: "Model",
       operators: textWithSuggestions(
-        topModels
+        allModelsData
```

Comment:
if topModels is not being used, let's delete it

Reply:
we still need topModels for graph, and need allModels for filters